templates/tfengine/components/terraform/main/main.tf

```diff
@@ -40,7 +40,7 @@ terraform {
     google-beta = ">= 3.0"
     {{end -}}
     {{if not $hasKubernetes -}}
-    kubernetes  = "~> 1.0"
+    kubernetes  = "~> 2.10"
```

Comment:
Is this because the new Terraform version no longer supports 1.0 provider?

Reply:
Yes.